### DISCUSSION TITLE
Using Github for web development that uses a database.

### DISCUSSION THREAD

**Discussion by Robbert-Cognito** [community/community#22927] [New to GitHub]
<p>HI,</p>
<p>I am doing research into how GitHub will work with our websites. Our websites use a database to store most of its content like text and links to images etc. My question is, how will Github work with this? Is there a way to also have version control of the databases? Or is there a 3rd party plugin/program which I need to use beside Github?</p>
<p>I hope my question is clear enough.</p>
<p>Kind regards,</p>
<p>Robbert</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/robbert-cognito">@robbert-cognito</a>,</p>
<p>Welcome to the GitHub community forums. :slight_smile:</p>
<p>GitHub is version control for files mostly — in terms of version control for databases, you could store database schema changes but the actual contents of the database is probably best backed up separately on a dedicated file service since it’ll be a hefty filesize that GitHub won’t necessarily cope well with.</p>
<p>That said, you <em>can</em> store and version large files on GitHub using Git Large File Storage - <a href="https://git-lfs.github.com/" rel="nofollow noopener">https://git-lfs.github.com/</a></p>
<p>To learn more about how GitHub works, I recommend reading the guides available at <a href="https://guides.github.com/" rel="nofollow noopener">https://guides.github.com/</a></p>
<p>​If there’s anything else I can help you with, let me know; so happy to help further!</p>
<p>Kindest regards,<br>
Matthew</p>

**Comment by Ndekere** [comment]
<p>Awesome! I love your github support… Keep up the good work.</p>
